packages/tocco-ui/src/utilStyles/declareDensity.js

```diff
@@ -1,9 +1,14 @@
 import objectToCss from './objectToCss'
 import {spaceScale} from './modularScale'
+import {stylingLook} from '../utilStyles'
 
 const declareDensity = props => objectToCss({
   'line-height': [props.dense ? 'lineHeights.dense' : 'lineHeights.regular'],
-  'padding': props.dense ? spaceScale(props, -4) : `${spaceScale(props, -3)} ${spaceScale(props, -2)}`
+  'padding': props.look === stylingLook.BALL
```

Comment:
I dont really have a advice ready right now but we have to be careful that these components dont tend to be swiss army knives with multiple if then cases inside.